### DISCUSSION TITLE
When will allocated value be reflected in the api?

### DISCUSSION THREAD

**Discussion by profd2004** [input-output-hk/catalyst-core#511] [General]
Testing out the testnet against fund 10 data and ProposalReview seems to be missing the `allocated` property from the API spec and `null` for all results coming from https://voices.projectcatalyst.io/api/proposal-reviews/ 

<img width="371" alt="allocated-missing" src="https://github.com/input-output-hk/catalyst-core/assets/4429661/4aa895f0-24fe-4efe-a8df-4de18e78d7fe">

Any clarity on this?

**Comment by coire1** [comment]
Hello Darlington!
The API that you're currently see exposed at `voices.projectcatalyst.io` it's not the Catalyst Data API currently deployed in the testnet, but it's a different one used by the moderation tool.
It includes only the subset of the information in the Catalyst Data API required for the moderation stage, and some additional functionalities to make it work (e.g. authentication).
Specifically, the fields `moderation`, `moderated`, `allocated` and `next_allocation`, are not database fields but dynamic properties returned to logged in user and generated by the backend.

When a user is logged in:
- `moderation` contains the current moderation submitted by logged user, if any
- `moderated` is `true` if the logged user submitted a moderation for that specific review
- `allocated` is `true` if the logged user has an allocation for that specific review. Only in that case the current user is allowed to submit a moderation.
- `next_allocation` includes the `id` of the next review allocated to the current user.

Hope this answer your question!
